### DISCUSSION TITLE
where the Spanish Programing Language?

### DISCUSSION THREAD

**Discussion by ricitos2001** [community/community#113470] [Other Feature Feedback, Questions, & Ideas]
EN: Why is there still no programming language adapted to the Spanish language?
ES: ¿Por qué todavía no existe un lenguaje de programación adaptado al idioma español?

**Comment by SebastienXia** [comment]
## Hello, @ricitos2001 

Read this comment [here](https://www.quora.com/Why-are-there-all-programming-languages-written-in-English-Why-not-other-languages-like-French-Spanish-and-Chinese) you will understand, why. 😉

**Reply by ricitos2001** [reply]
ok

**Comment by fabianlanza** [comment]
ES: Hola, personalmente con companeros hemos tenido esta misma pregunta. Todos opinamos y llegamos a la conclusion que es porque Ingles es un lenguage que se considerar Internacional. A que me refiero con esto, pues que si notas no importa a que pais vayas siempre hay muchas personas que hablan ingles. Si viajas Ingles es el lenguage que probablemente puedas usar sin importal el lenguage natal del pais donde estes.

EN: Hello, personally with colleagues we have had this same question. We all gave our opinion and came to the conclusion that it is because English is a language that is considered International. What do I mean by this, because if you notice, no matter what country you go to, there are always many people who speak English. If you travel, English is the language you can probably use regardless of the native language of the country where you are.

**Reply by ricitos2001** [reply]
si lo miro desde ese punto de vista tiene sentido que el lenguaje de programación sea en ingles

**Reply by IOrlandoni** [reply]
![https://translate.how/translate.png](http://translate.how/en/spanish/)

**Comment by git-sumana** [comment]
English language being dominant throughout the world, most of the languages are developed using EN, there is no specific reason the avoid Spanish language to my knowledge

**Reply by ricitos2001** [reply]
im understand

**Reply by git-sumana** [reply]
yeah good to know!
